client/webserver/http.go

```diff
@@ -302,7 +344,6 @@ func (s *WebServer) handleExportOrders(w http.ResponseWriter, r *http.Request) {
 			log.Errorf("error writing CSV: %v", err)
 			return
 		}
-		wf.Flush()
```

Comment:
ACK.  Just reiterating that I don't think there's a need to `Flush` the `http.ResponseWriter` at the end of an `http.Handler`, so good to remove these.   Flushing the csvWriter on the other hand make sense to me, and that's done.